src/tracks/InstrumentTrack.cpp

```diff
@@ -713,9 +713,11 @@ bool InstrumentTrack::play( const MidiTime & _start, const fpp_t _frames,
 
 
 
-TrackContentObject * InstrumentTrack::createTCO( const MidiTime & )
+TrackContentObject * InstrumentTrack::createTCO( const MidiTime & _pos )
```

Comment:
Sorry that I didn't highlight it earlier, but the same (lack of) spacing should apply to the parameter list. Feel free to disregard, I don't think something this minor should hold back the PR.

Reply:
I fixed the spacing on the parameters list and on `BBTrack::createTCO` even on the statements that weren't added by this PR